packages/stockflux-launcher/src/search-results/Launcher.js

```diff
@@ -33,16 +32,19 @@ export default async (
     showTaskbarIcon: false,
     backgroundColor: '#28313D',
     waitForPageLoad: true,
-    alwaysOnTop: true
+    alwaysOnTop: false,
+    maxWidth: 400,
+    minWidth: 400,
+    minHeight: 300
   };
 
   const win = await OpenfinApiHelpers.createWindow(childWindow);
 
-  if (dockedTo === ScreenEdge.LEFT || dockedTo === ScreenEdge.RIGHT) {
-    win
-      .getWebWindow()
-      .document.getElementById('searchbar-container').hidden = false;
-  }
+  // if (dockedTo === ScreenEdge.LEFT || dockedTo === ScreenEdge.RIGHT) {
```

Comment:
Does this commented out code need to be deleted?

Reply:
@kangelSL, you asked about the bit where you click away and the search remains open and if the window should close and the search bar clear. This PR does not add additional functionality, but rather introduces useChildWindow hook and ensured the launch buttons in the child window work. 

Regarding the second question, I think @jrhenderson1988 answered this perfectly and I have nothing to add! :)

> A couple of small things, but I was also wondering:
> 
> * When clicking away, the search remains open. Should the window close and the search bar clear if they click away?
> * Is the search case sensitive? E.g. if a company has an all-caps name would it appear first in the list if searched for using all-caps?